include/picongpu/plugins/radiation/Radiation.kernel

```diff
@@ -363,6 +355,7 @@ namespace picongpu
                                 /* storage for amplitude (complex 3D vector)
                                  * it  is initialized with zeros (  0 +  i 0 )
                                  */
+                                // ACCUMULATOR! Ensure double precision
```

Comment:
please move this comment into the comment above

Reply:
Done in 1007070